static-files/ocp_on_premise.yaml

```diff
@@ -0,0 +1,178 @@
+---
+generators:
+  - OCPGenerator:
+      start_date: 2019-05-01
+      end_date: 2019-07-31
+      nodes:
+        - node:
+          node_name: compute_1
+          cpu_cores: 4
+          memory_gig: 16
+          namespaces:
+            Toronto:
```

Comment:
We can add additional entries under namespaces for each city name we want to see as a namespace/project.